quickstarts/enable-developer-perspective.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleQuickStart
+metadata:
+  name: enable-developer-perspective
+spec:
+  conclusion: You have learned how to enable and disable the Developer perspective!
+  description: Learn how to enable and disable the Developer perspective.
+  displayName: Enable the Developer Perspective
+  durationMinutes: 5
+  introduction: 'This guide walks you through enabling the Developer perspective in OpenShift. You will learn how to modify console setting via the web interface or the CLI, allowing developers to access a streamlined, developer-friendly view within the OpenShift web console.'
+  prerequisites:
+    - You must have cluster administrator or appropriate RBAC permissions to modify console settings.
+  tags:
+    - developer
+    - perspective
+    - ODC
+  tasks:
+    - description: |-
+        **Search page**
+        1. Open the **Home** section in the side navigation, and navigate to the **Search** page
+      review:
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+        instructions: |-
+          To verify that you are in the right place:
+          1. You should see a search bar and a Resource filter dropdown at the top of the page.
+      summary:
+        failed: Try the steps again.
+        success: Great work!
+      title: Navigate to the search page
+    - description: |-
+        **Search results**
+        1. Under the **Resource** filter dropdown, search for **operator.openshift.io/v1**
+      review:
```

Comment:
If I search `operator.openshift.io/v1` in resource search bar, I am not getting any result.

<img width="1454" alt="image" src="https://github.com/user-attachments/assets/a1199ca6-bdc3-428f-8710-20a2fc44606b" />

Reply:
Updated the steps.